src/record_syscall.cc

```diff
@@ -4598,8 +4598,24 @@ static Switchable rec_prepare_syscall_arch(RecordTask* t,
               syscall_state.expect_errno = EINVAL;
               break;
           }
+          break;
+        }
+
+        case PR_SET_VMA: {
+          switch (regs.arg2()) {
+            case PR_SET_VMA_ANON_NAME:
+              // PR_SET_VMA_ANON_NAME is used by Android to communicate
```

Comment:
nit: The Android reference here seems somewhat misleading since this is in mainline now.